addons/mail/static/src/components/attachment/attachment_tests.js

```diff
@@ -595,6 +595,47 @@ QUnit.test('close attachment viewer', async function (assert) {
     );
 });
 
+QUnit.test('delete an attachment', async function (assert) {
```

Comment:
test title should describe more closely the issue, suggestion: "clicking on the delete attachment button multiple times should do the rpc only once"